implementations/rust/ockam/ockam_node/Cargo.toml

```diff
@@ -10,11 +10,31 @@ license = "Apache-2.0"
 name = "ockam_node"
 version = "0.23.0-dev"
 
+[features]
+default = ["std"]
+
+# Option (enabled by default): "std" enables functionality expected to
+# be available on a standard platform.
+std = ["ockam_core/std", "tokio", "tracing-subscriber"]
+
+# Option: "no_std" enables functionality required for platforms
+# without the standard library.
+no_std = ["ockam_core/no_std", "ockam_node_no_std/no_std", "heapless"]
+
+# Option: "alloc" enables support for heap allocation on "no_std"
+# platforms, requires nightly.
+alloc = ["ockam_core/alloc", "ockam_node_no_std/alloc"]
+
+# Option: "dump_internals" when set, will dump the internal state of workers at startup via the trace! macro.
+dump_internals = []
```

Comment:
I think we don't use it anymore

Reply:
I don't remember why I removed it, but if it works, we can keep it

Reply:
It's still in `ockam_node/src/router.rs` afaik.

Reply:
I think I removed it during my latest changes to node. But if it works, we can keep it